tst/client/StreamPutGetTest.cpp

```diff
@@ -1427,8 +1427,8 @@ TEST_F(StreamPutGetTest, putFrame_PutGetNotifyAndTagsStoreData)
         // Set the frame bits
         MEMSET(frame.frameData, (BYTE) i, SIZEOF(tempBuffer));
 
-        // Key frame every 10th
-        frame.flags = i % 10 == 0 ? FRAME_FLAG_KEY_FRAME : FRAME_FLAG_NONE;
```

Comment:
Would like to be sure there isn't a reason we had it at 10 frames per fragment, but hard to know without OC's input. If there is a reason and we want to keep the 10 frames/fragment, can skip the first 2 frames from getting metadata added.

Reply:
From the test it doesn't seem like there was a reason for that. Previously there were 10 key frames and each fragment was allowed 25 metadata tags max. Now, it is 8 frames per fragment and they're allowed 10 metadata tags. I had to keep it 8 because putEventMetadata adds 2 tags per call and if the keyframe is every 10th frame, we exceed the max tags which is 10 now